news/2019-04-08-project-loved-week-of-april-7th.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title: "Project Loved: Week of April 7th"
+date: 2019-04-08 03:00:00 +0000
+---
+
+Every second Sunday is [Project Loved](https://osu.ppy.sh/community/forums/120) day! We've got a new selection of maps for you to try, and now it's your turn to decide whether or not they should enter the Loved beatmap category!
+
+![](/wiki/shared/news/banners/project-loved.jpg)
+
+Welcome back to [Project Loved](https://osu.ppy.sh/community/forums/120)! Every two weeks, we pick out a few beatmaps from each game mode that will get their shot at making it into the [Loved category](https://osu.ppy.sh/beatmapsets?s=8). From the maps we selected, you can now vote whether or not you want them to be Loved! Click on each map's header image in this post to be taken to its voting thread.
+
+Our osu!mania friends interested in Project Loved are probably all aware of the unfortunate situation last round with Mat's Architecture voting (see [the last post on its forum thread](https://osu.ppy.sh/community/forums/posts/7031962) if you aren't caught up!). To help combat illegitimate votes, bandwagoning, and other related issues, we've implemented a measure to keep the voting results hidden until the polls are completed. We may look into other solutions if these problems occur again.
+
+In case you don't read the end of this post, be sure to submit suggestions for beatmaps to be added to Loved using the [Google form](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)! All of the nominated beatmaps are chosen from submissions on this form, which you can view in the corresponding [Google sheet](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit). For osu!mania, please use [this Google form](https://docs.google.com/forms/d/e/1FAIpQLSeaGfoQNGMqw4qQcqRPItUZILh2fGwJR6ly6cZNY9OWPXkFhw/viewform) and [this Google sheet](https://docs.google.com/spreadsheets/d/1sjkTwUSvQ5Me-6rK61rToTg2bU-yX9X29CXdzttvhtM/edit) instead.
+
+### Navigation
+
+- [osu!](#osu)
+- [osu!taiko](#osutaiko)
+- [osu!catch](#osucatch)
+- [osu!mania](#osumania)
+
+## <a name="osu" id="osu"></a>osu!
+
+osu! Loved candidates were chosen by [DigitalHypno](https://osu.ppy.sh/users/4384207), [-Roxas](https://osu.ppy.sh/users/1986262), [BeasttrollMC](https://osu.ppy.sh/users/3171691), [Bubbleman](https://osu.ppy.sh/users/5182050) and [waywern2012](https://osu.ppy.sh/users/5870453)!
```

Comment:
candidates -> picks?

Reply:
they mean the same thing here i just use synonyms sometimes cuz it appears frequently in this post
"nominations" "selections" etc. too